src/afa/afa.cc

```diff
@@ -311,7 +872,8 @@ void Mata::Afa::construct(
       formula += body_line[i] + " ";
     }
 
-    aut->add_trans(src_state, formula);
+    //aut->add_trans(src_state, formula);
+    //TODO
```

Comment:
Again here. What are we supposed to do? Why to keep the commented out line here? Try to resolve it.

Reply:
Thanks for your suggestion, I have described the issue in the latest version of the code.